src/ripple/ledger/ReadView.h

```diff
@@ -38,17 +40,41 @@
 
 namespace ripple {
 
+// Tag types to differentiate units
+struct DropsTag;
```

Comment:
Seems like these aliases (`Drops32`, `Drops64`, etc.) could be used in a few places (I saw the fwd declares for them in places). Why not stick them in a header to reuse?